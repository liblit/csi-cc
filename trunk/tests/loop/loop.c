#include <stdio.h>

int rand(){
  static int x = 3; 
  return(x = (x * 8121 + 28411) % 134455);
}

int main(){
  int x = rand()%14;
  while(x!=2){
    printf("ANSWER: %d\n", x);
    int y = rand()%2;
    if(y==1){
      printf("Y= %d\n", 1);
    }
    else
      printf("Y= %d\n", 0);
    x = rand()%14;
  }
  printf("DONE: %d\n", x);
}
