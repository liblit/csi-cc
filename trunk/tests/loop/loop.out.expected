ANSWER: 8
Y= 0
ANSWER: 9
Y= 1
ANSWER: 0
Y= 1
DONE: 2
